src/main/java/uk/gov/ch/pocappointmentapi/Appointment.java

```diff
@@ -0,0 +1,226 @@
+package uk.gov.ch.pocappointmentapi;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Document(collection="appointments")
+public class Appointment {
```

Comment:
Could use Hibernate OGM with an EntityManager and JPA annotations to make this more portable.

Reply:
True, but given this just a POC, I don't think portability is a requirement.